include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,102 @@
+#ifndef _NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_
+#define _NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_
+
+namespace nbl::video
+{
+
+struct SPhysicalDeviceFeatures
+{
+    /* Vulkan Core 1.0 */
+    bool robustBufferAccess = false;
+    //VkBool32    fullDrawIndexUint32;
+    bool imageCubeArray = false;
+    //VkBool32    independentBlend;
+    bool geometryShader    = false;
+    //VkBool32    tessellationShader;
+    //VkBool32    sampleRateShading;
+    //VkBool32    dualSrcBlend;
+    bool logicOp = false;
+    bool multiDrawIndirect = false;
+    //VkBool32    drawIndirectFirstInstance;
+    //VkBool32    depthClamp;
+    //VkBool32    depthBiasClamp;
+    //VkBool32    fillModeNonSolid;
+    //VkBool32    depthBounds;
+    //VkBool32    wideLines;
+    //VkBool32    largePoints;
+    //VkBool32    alphaToOne;
+    bool multiViewport = false;
+    bool samplerAnisotropy = false;
+    //VkBool32    textureCompressionETC2;
+    //VkBool32    textureCompressionASTC_LDR;
+    //VkBool32    textureCompressionBC;
+    //VkBool32    occlusionQueryPrecise;
+    //VkBool32    pipelineStatisticsQuery;
+    //VkBool32    vertexPipelineStoresAndAtomics;
+    //VkBool32    fragmentStoresAndAtomics;
+    //VkBool32    shaderTessellationAndGeometryPointSize;
+    //VkBool32    shaderImageGatherExtended;
+    //VkBool32    shaderStorageImageExtendedFormats;
+    //VkBool32    shaderStorageImageMultisample;
```

Comment:
want to expose this
https://github.com/Devsh-Graphics-Programming/Nabla/pull/348#discussion_r867400431

Reply:
yes

Reply:
`shaderStorageImageMultisample` just assume always true on GL and GLES (bot sure of the GLES, but ESSL 3.10 seems to bless this)

Reply:
>  (bot sure of the GLES, but ESSL 3.10 seems to bless this) 
didn't get this part

Reply:
ESSL 3.10 spec (the one that comes with GLES 3.1) seems to allow `image2Dms`